content/en/docs/tasks/administer-cluster/running-cloud-controller.md

```diff
@@ -46,6 +46,13 @@ integration, it should not be too different from the requirements when running
 
 Successfully running cloud-controller-manager requires some changes to your cluster configuration.
 
+  {{< note >}}
+  The `--cloud-provider` flag is used differently for `kubelet`, `kube-apiserver` and
+  `kube-controller-manager` depending on the user. If the user has an external CCM
+  (not the internal cloud controller loops in the Kubernetes Controller Manager), then
+  `--cloud-provider=external` must be specified. Otherwise, it should not be specified.
+  {{< /note >}}
+
 * `kube-apiserver` and `kube-controller-manager` MUST NOT specify the `--cloud-provider`
```

Comment:
perhaps something like

```
* `kube-apiserver` and `kube-controller-manager` must be set according to the user's usage of external CCM. If the user has an external CCM (not the internal cloud controller loops in the Kubernetes Controller Manager), then `--cloud-provider=external` must be specified. Otherwise, it should not be specified.
```

Reply:
Hi @elmiko,
Thanks for making understand.
[Above](https://github.com/kubernetes/website/pull/43010#discussion_r1337250155) it mention only about `kube-apiserver` and `kube-controller-manager`. What about kubelet that mention in the second bullet point. Do i need to replace that also?

> kubelet must run with --cloud-provider=external. This is to ensure that the kubelet is aware that it must be initialized by the cloud controller manager before it is scheduled any work.

Please suggest.

Reply:
i think we could combine these 2 point together, we want to tell users that whenever they use the external CCM then they should set `--cloud-provider=external`.  maybe we could say:

```
* `kubelet`, `kube-apiserver` and `kube-controller-manager` must be set according to the user's usage of external CCM. If the user has an external CCM (not the internal cloud controller loops in the Kubernetes Controller Manager), then `--cloud-provider=external` must be specified. Otherwise, it should not be specified.
```

then we could remove the second point from the list.

Reply:
Yes, That is what i am also expecting. Will follow the same.

Reply:
awesome, thanks @T-Lakshmi !

Reply:
Updated as per the discussion.
PTAL!